_posts/EternalFootman/2016-02-09-more-8.md

```diff
@@ -0,0 +1,16 @@
+---
+layout: post
+author:
+-EternalFootman
+-camazotz
+title: "Kit and Navaneet's Chapter 8 Excercises"
+---
+
+8.4:
+<iframe src="https://trinket.io/embed/python/20dd6e17ca?start=result" width="100%" height="600" frameborder="0" marginwidth="0" marginheight="0" allowfullscreen></iframe>
+
+8.5:
+<iframe src="https://trinket.io/embed/python/85fe792850?start=result" width="100%" height="600" frameborder="0" marginwidth="0" marginheight="0" allowfullscreen></iframe>
+
+8.6:
+<iframe src="https://trinket.io/embed/python/7ca71bfd70?start=result" width="100%" height="600" frameborder="0" marginwidth="0" marginheight="0" allowfullscreen></iframe>
```

Comment:
The idea on this one was to build a list from the input and then use the `min` and `max` functions to do the hard work you're doing with all those comparisons. Make sure you understand how to do that!  Take a look at some other groups' work.